bin/createproject.sh

```diff
@@ -324,15 +328,30 @@ git commit -m 'First commit - project started'
 git push -u origin --all
 
 echo "server {
-    listen 80;
-    #listen [::]:80 default ipv6only=on;
-
+    listen 443 ssl http2;
     root /var/www/$PROJECTNAME;
-    index index.html index.htm index.php;
-
+    index index.php;
     server_name $PROJECTNAME.test www.$PROJECTNAME.test;
+
     include php7.conf;
     include global/wordpress.conf;
+
+    ssl_certificate /var/www/certs/$PROJECTNAME.test.pem;
+    ssl_certificate_key /var/www/certs/$PROJECTNAME.test-key.pem;
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+    ssl_prefer_server_ciphers on;
+    ssl_dhparam /etc/ssl/certs/dhparam.pem;
```

Comment:
It looks like `mkcert` does not automatically generate `dhparam.pem` nor our script do check that it's there. In case the user hasn't generated it, nginx wont start.

In fact, I don't think that dhparam is necessary for dev env. Thought it's good practice to have that, so maybe worth adding a check that generates it if not already done, or at least note on readme that you'll need to create a one?

```
cd /etc/ssl/certs
openssl dhparam -out dhparam.pem 4096 
```

Reply:
I think [it is necessary](https://stackoverflow.com/questions/46463964/is-ssl-dhparam-necessary-for-nginx-when-client-use-the-cipher-dhe-rsa-aes256-sha). IIRC it's generated when nginx/system packages are installed, can't really remember.